src/components/MainRouter/MainRouter.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Switch, Route } from "react-router-dom";
+import MainNavigation from "../MainNavigation/MainNavigation";
+
+const MainRouter = () => {
+  return (
+    <div>
```

Comment:
In order to prevent using a lot of divs you can also easily wrap this content to `<></>` (of course if it doesn't affect your layout)